src/core/Rakudo/Internals.pm6

```diff
@@ -321,6 +321,8 @@ my class Rakudo::Internals {
       # ShiftJIS
       'windows-932',     'windows-932',
       'windows932',      'windows-932',
+      # GB2312
+	  'gb2312',          'gb2312',
```

Comment:
This line is misindented (tabs vs spaces?).

Reply:
Agreed. Just fix the alignment of this one.